test/readwrite.jl

```diff
@@ -0,0 +1,72 @@
+@testset "Read and write" begin
+    import ONNX
+
```

Comment:
@dfdx I'm wondering if these tests are enough. Don't we also need operator level tests? Basically tests for our mappings of https://github.com/onnx/onnx/blob/master/docs/Operators.md

Reply:
This PR is more level thing so far. The details can be found on [the discussion](https://github.com/FluxML/ML-Coordination-Tracker/discussions/24), but the main points are:

* this package in its current state is essentially unmaintained
* there are several competing ONNX libraries out their used in different contexts
* it would be great to have a single implementation - preferably here - that works not only for Flux, but also for other projects (e.g. [Avalon.jl](https://github.com/dfdx/Avalon.jl) which I maintain)
* we don't have agreement on the high-level API yet (e.g. ONNX operators)
* but there's a set of low-level utilities for reading and writing ONNX's ProtoBuf definitions
* the plan is to settle these low-level utilities first and based on it move our discussion of operators further

As you can see from the code, there are only convenient methods for reading and writing ONNX entities without the need to dive into all the subtleties of the format specification. This is also the reason why there are no proper usage examples - it is simply not designed for the end users (yet). 

I realize this is a huge change, but since this package doesn't work with the latest Flux version anyway, I think it's fair to step back and redesign everything almost from scratch. Yet I'll be happy to see alternative approaches to the problem.

Reply:
Got it.